Twig/Extension/DeprecatedTextExtension.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\CoreBundle\Twig\Extension;
+
+/**
+ * NEXT_MAJOR : remove this class and the twig/extensions dependency.
```

Comment:
I don't like being unsure, so let's discuss this here : https://github.com/sonata-project/dev-kit/issues/206